components/molecule/pagination/src/index.js

```diff
@@ -19,7 +18,11 @@ const PageButton = ({onSelectPage, page, design, ...props}) => {
   const _onSelectPage = e => {
     onSelectPage(e, {page})
   }
-  return <AtomButtom onClick={_onSelectPage} design={design} {...props} />
+  return (
+    <li className={`${BASE_CLASS}-item`}>
+      <AtomButtom onClick={_onSelectPage} design={design} {...props} />
```

Comment:
shouldn't this be AtomButto**n** (n) instead of AtomButto**m** ?

Reply:
it seems a misspelling...

Reply:
merci!